src/ops/ConfigOps.ts

```diff
@@ -142,7 +142,7 @@ export async function exportEverythingToFiles(
  * @param {boolean} extract Extracts the scripts from the exports into separate files if true
  * @param {boolean} separateMappings separate sync.idm.json mappings if true, otherwise keep them in a single file
  */
-function exportItem(
```

Comment:
Rather than exporting this, I would refactor the exportEverythingToFiles to take in the export directory as a parameter, that, since everything else is the same in the PromotionOps version